### PR TITLE
vl3-nse: remove client facing interfaces

### PR DESCRIPTION
**Pull request by CosminNechifor**
This PR add functionality for the removal of client facing interfaces when the ucnf connection close handler is called.

Closes: cisco-app-networking/cnns-prj#151

### CODE REVIEW

File: pkg/universal-cnf/config/composite.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ package config
 
 import (
 	"context"
-
+	"fmt"
```
Comment (danielvladco) [inline pkg/universal-cnf/config/composite.go:20]: sort imports

File: pkg/universal-cnf/config/composite.go [hunk 2]
```diff
@@ -62,10 +64,87 @@ func (uce *UniversalCNFEndpoint) Request(ctx context.Context,
 	return request.GetConnection(), nil
 }
 
+// Removes the client interfaces from the dpConfig and
+// Returns a new *vpp.ConfigData which contains the removed interfaces.
+func (uce *UniversalCNFEndpoint) removeClientInterface(connection *connection.Connection) (*vpp.ConfigData, error) {
```
Comment (danielvladco) [inline pkg/universal-cnf/config/composite.go:69]: we should add a test for this method to make sure the remove logic works correctly

**Review by rastislavs — APPROVED** [review]
LGTM

**Review by ana-moga — APPROVED** [review]
Cool this is fixed. Lgtm

File: pkg/universal-cnf/config/composite.go (hunk 2 above)
Comment (tiswanso) [reply pkg/universal-cnf/config/composite.go:69]: @danielvladco -- good point.  Given there's no other UT type tests for the ucnf/config/composite.go, I'm adding an issue for that as a work item for a follow-on PR.

**Review by tiswanso — APPROVED** [review]

**Pull request merged**